.github/CODEOWNERS

```
@@ -1 +1 @@
-* @sanderland @cognitedata/starbase
+@cognitedata/starbase
```

```suggestion
* @cognitedata/starbase
```
not sure if omitting * works